packages/forklift-console-plugin/tsconfig.json

```diff
@@ -1,22 +1,25 @@
 {
   "extends": "../../config/tsconfig.base.json",
 
-  "include": ["src/**/*", "*.ts"],
+  "include": ["./src/**/*", "./plugin-metadata.ts", "./plugin-extensions.ts"],
```

Comment:
Missing the include for `webpack.config.ts`?

Reply:
I was conflicted about that too, do we need to include this configuration in the tsconfig that define how we compile our source code ? my reasoning was that it is not part of the source, do you see a reason to include it ?

Reply:
Humm...I don't know what I did before, but your current incantations on `tsconfig.json` is working fine.